model/artifact/artifact_file.go

```diff
@@ -88,6 +94,18 @@ func GetAllArtifacts(tasks []TaskIDAndExecution) ([]File, error) {
 	}
 	files := []File{}
 	for _, artifact := range artifacts {
+		for i := range artifact.Files {
+			if artifact.Files[i].Visibility == "signed" {
+				if artifact.Files[i].AwsSecret == "" || artifact.Files[i].AwsKey == "" || artifact.Files[i].Bucket == "" || artifact.Files[i].FileKey == "" {
```

Comment:
can you make this if statement a method on the file type?

Reply:
Do you mean to add a method to artifact_file.go that validates that the values are there?

Reply:
I'm unsure what you mean by "file type".